browser-test/src/admin_program_list.test.ts

```diff
@@ -173,12 +173,12 @@ describe('Program list page.', () => {
     await adminPrograms.addQuestionFromQuestionBank(nameQuestion)
 
     await adminPrograms.gotoAdminProgramsPage()
-    await page.click('#program-one-publish-modal-button')
-    expect(await page.innerText('#program-one-publish-modal')).toContain(
+    await page.click('#publish-modal-program-one-button')
```

Comment:
Non-blocking: Consider putting all the various `#publish-modal-xxx` and `#publish-modal-xxx-button` strings into a variable.

Reply:
done